src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java

```diff
@@ -16,11 +16,12 @@
 
 public class TestAdsTrackingPixels extends NewTestTemplate {
 
-  public static final String COMSCORE_PIXEL_URL = "http://b.scorecardresearch.com/b";
-  public static final String GA_PIXEL_URL = "https://www.google-analytics.com/collect";
-  public static final String KRUX_PIXEL_URL = "https://beacon.krxd.net/pixel.gif";
-  public static final String QUANTQAST_PIXEL_URL = "http://edge.quantserve.com/";
-  public static final String QUANTQAST_PIXEL_URL_SECURE = "https://pixel.quantserve.com/";
+//  public static final String COMSCORE_PIXEL_URL = "http://b.scorecardresearch.com/b";
+//  public static final String GA_PIXEL_URL = "https://www.google-analytics.com/collect";
+//  public static final String KRUX_PIXEL_URL = "https://beacon.krxd.net/pixel.gif";
+private static final String ADS_QUANTCAST_PATTERN = "http?://.*quantserve\\.com.*";
+  public static final String QUANTQAST_PIXEL_URL = "http?://edge\\.quantserve\\.com.*";
+//  public static final String QUANTQAST_PIXEL_URL_SECURE = "https://pixel.quantserve.com/";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)